config/locales/views.en.yml

```
@@ -270,7 +270,8 @@ en:
       help:
         password: "Leave this blank if you don't want to change it"
       delete:
-        action: "Delete my account"
+        action: "Permanently delete account"
+        confirm: "Are you sure? This deleting your account is IRREVERSIBLE."
```

This deleting is the
```suggestion
        confirm: "Are you sure? Deleting your account is IRREVERSIBLE."
```